.github/workflows/release.yml

```diff
@@ -49,15 +49,10 @@ jobs:
       - name: Build 'kolena' Python package
         run: poetry build --format=sdist
 
-      - name: Build 'kolena' documentation and push to GitHub Pages
+      - name: Publish 'kolena' documentation to production (S3)
         run: |
-          git config user.name 'kolenabot'
-          git config user.email 'bot@kolena.io'
-          ./docs/setup_insiders.sh
-          poetry run mkdocs gh-deploy --verbose --strict --remote-branch release/docs --config-file mkdocs.insiders.yml
-        env:
```

Comment:
Are these vars not still necessary? We still have DD RUM plugged into the docs.

Reply:
We still have it in the mkdocs config [here](https://github.com/kolenaIO/kolena/blob/de651a6f7e4ad548c5bd0796da3df60d41e010dd/mkdocs.yml#L215-L217)

Reply:
If they're not mounted in these CI steps, how are they getting populated? I see `blank` tokens on docs.kolena.io currently.

Reply:
How are they used? can we also remove that?

Reply:
seems this is related to datadog

Reply:
asking around in general channel